src/app/login-page/components/login-page/login-page.component.ts

```diff
@@ -10,9 +10,45 @@ import { Observable } from 'rxjs';
 })
 
 export class LoginPageComponent implements OnInit {
+  cases:any;
   constructor(
-  ) {}
+  ) {
+
+  }
 
   ngOnInit(): void {
+
+    this.cases = [
```

Comment:
This is a good start. 
But in the ticket on trello, it has some info on where you can get the data from the API
https://trello.com/c/6lTcfYaf/7-login-page-solved-cases-list

So could you switch it to using the real data?